src/nvhttp.cpp

```diff
@@ -683,13 +683,22 @@ void applist(resp_https_t response, req_https_t request) {
 
   apps.put("<xmlattr>.status_code", 200);
 
-  int x = 0;
-  for(auto &proc : proc::proc.get_apps()) {
+  for(int i = 0; i < proc::proc.get_apps().size(); i++) {
+    auto &proc = proc::proc.get_apps()[i];
```

Comment:
The original for each loop was more efficient since it avoids 2 calls to `proc.get_apps()` each iteration.